File: dayFour/python/After Break/task_seven.py
total = 0
average = 0
count = 0
for index in range(10):
    number = int(input("Enter a number: "))
    if number % 2 == 0: 
        total += number
        count += 1

average = total / count
print("sum is: ", total)
print("average is: ", average)

File: dayFour/javaScript/After Break/TaskSeven.js
const prompt = require("prompt-sync")();
let sum = 0;
let average = 0;
let count = 0;
for(let index = 1;  index <= 10; index++){
    let number = Number(prompt("Enter a number: "));
    if (number % 2 == 0) {
        sum += number;
        count++;
    }
}
average = sum / count;
console.log("sum is: ", sum);
console.log("average is: ", average);

